### ISSUE TITLE
Failed to install with the node v11.12.0

### ISSUE THREAD

**Issue by ender503** [digitalbazaar/crypto-ld#32]
I got these errors when I install the `crypto-ld` with node v11.12.0, because of the `engines` field specifies the node version must be "8.3.0".

Could I send the pull request to change it to `{ "engines" : { "node" : ">=8.3.0" } }`? 😃 

```bash
$ yarn add crypto-ld
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error crypto-ld@2.0.1: The engine "node" is incompatible with this module. Expected version "^8.3.0". Got "11.12.0"
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by mattcollier** [comment]
Hello, sorry for the inconvenience, I'll fix this now.

**Comment by mattcollier** [comment]
Thanks for the report.  Landed fix in v3.5.2 https://github.com/digitalbazaar/crypto-ld/commit/fc38c3f859fd058a99f56e13ee288b521f0e6bc6

**Comment by ender503** [comment]
Thanks for your help 👍
